documentation/qos_monitor/sdd/qos_monitor_sdd.md

```diff
@@ -156,6 +157,97 @@ GET /qos_monitor/echo
+| ------------------------------------------------------- |
+
+| Field | Description | Necessity | Format/Limitations |
+| ----- | ----------- | --------- | ----------- |
+| `eventType` | Type of event. | mandatory | must be parsable to a valid [QosMonitorEventType](#input_qos_monitor_event_type) |
+| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key associated with a string value, which is parsable to UUID object |
+| `payload` | String representation of the event. | mandatory | must be an empty list as "{[]}", unless the event type is FINISHED_MONITORING_MEASUREMENT, otherwise it must be a list of [IcmpPingResponse](#input_icmp_ping_response) |
+| `timestamp` | The time of publishing  | mandatory | UTC time in `yyyy-MM-dd` `T` `HH`:`mm`:`ss.sss` `Z` format |
+
+**QosMonitorEventType:** <a name="input_qos_monitor_event_type"/>
+
+
+| __RECEIVED_MONITORING_REQUEST__  type |
+| ------------------------------------------------------- |
+
+| Field | Description | Necessity | Format/Limitations |
+| ----- | ----------- | --------- | ----------- |
+| `eventType` | Type of event. | mandatory | must be "RECEIVED_MONITORING_REQUEST" |
+| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key associated with a string value, which is parsable to UUID object |
+| `payload` | String representation of the event. | mandatory | must be an empty list as "{[]}" |
+| `timestamp` | The time of publishing  | mandatory | UTC time in `yyyy-MM-dd` `T` `HH`:`mm`:`ss.sss` `Z` format |
+
+
+| __STARTED_MONITORING_MEASUREMENT__  type |
+| ------------------------------------------------------- |
+
+| Field | Description | Necessity | Format/Limitations |
+| ----- | ----------- | --------- | ----------- |
+| `eventType` | Type of event. | mandatory | must be "STARTED_MONITORING_MEASUREMENT" |
+| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key associated with a string value, which is parsable to UUID object |
+| `payload` | String representation of the event. | mandatory | must be an empty list as "{[]}" |
+| `timestamp` | The time of publishing  | mandatory | UTC time in `yyyy-MM-dd` `T` `HH`:`mm`:`ss.sss` `Z` format |
+
+| __INTERRUPTED_MONITORING_MEASUREMENT__  type |
+| ------------------------------------------------------- |
+
+| Field | Description | Necessity | Format/Limitations |
+| ----- | ----------- | --------- | ----------- |
+| `eventType` | Type of event. | mandatory | must be "INTERRUPTED_MONITORING_MEASUREMENT" |
+| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key associated with a string value, which is parsable to UUID object, must contain an "exception" key associated with a string value,  may contain an "root_cause" key associated with a string value|
```

Comment:
According to the QoSMonitorConstants, the keys must be EXCEPTION and ROOT_CAUSE, not the lower-case version

Reply:
done